.evergreen/config/functions.yml

```diff
@@ -242,12 +256,6 @@ functions:
           # Don't use ${file} syntax here because evergreen treats it as an empty expansion.
           [ -f "$file" ] && sh $file || echo "$file not available, skipping"
 
-  "decrypt atlas URI list":
-    - command: shell.exec
-      params:
-        script: |
-          openssl aes-256-cbc -S "${encrypted_uris_salt}" -K "${encrypted_uris_key}" -iv "${encrypted_uris_iv}" -in ${PROJECT_DIRECTORY}/.evergreen/atlas-uris.txt.enc -out ${PROJECT_DIRECTORY}/.evergreen/atlas-uris.txt -d
```

Comment:
> This makes it available for us to use in tests without having to decrypt a file and potentially leak the encryption credentials.

Are other drivers doing this as well? I don't have an opinion on which is more secure, but storing the URIs directly in the Evergreen project config is certainly more convenient.

Note: you can also remove `.evergreen/atlas-uris.txt` from `.gitignore`.